templates/deploy-ansible-service-broker.template.yaml

```diff
@@ -418,7 +418,7 @@ parameters:
 - description: Container Image to use for Ansible Service Broker in format of imagename:tag
   displayname: Ansible Service Broker Image
   name: BROKER_IMAGE
-  value: ansibleplaybookbundle/origin-ansible-service-broker:latest
+  value: ansibleplaybookbundle/origin-ansible-service-broker:release-1.0
```

Comment:
Do we just do this as part of our branching exercise?

Reply:
I believe we should. I think in the future, when we talk about branching, we should make sure the container images (and tags) exist for the branch and are up to date then create the new branch with these changes.